### ISSUE TITLE
The user profile is detected automatically

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#11]
## User story
1. As a user that reports a problem
2. I don't want give my contact details explicitly 
3. So that I don't have to waste time inputting this information again and again

## Acceptance criteria
* For calls and Emails I get the creation notification without ever telling the system my contact information
* For the WebApp after giving my contact information to the system once, I don't ever have to do it again.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
